pkg/utils/syscalls/syscalls-helper.go

```diff
@@ -0,0 +1,46 @@
+package syscalls
```

Comment:
nit: please don't forget to add the license header as we have it on other files.

Reply:
Done